testing/cuda/uninitialized_fill.cu

```diff
@@ -163,9 +163,6 @@ void TestUninitializedFillNDevice(ExecutionPolicy exec)
     ASSERT_EQUAL(cudaSuccess, err);
   }
 
-  cudaError_t const err = cudaDeviceSynchronize();
-  ASSERT_EQUAL(cudaSuccess, err);
```

Comment:
Why remove this?

Reply:
It's redundant and has identifier conflicts with the block scope that precedes it.